### PR TITLE
fix(test): updated uuid in kyverno validation

### PR DESCRIPTION
**Pull request by CloudBeard**
## Description

The Kyverno validation UUID in the dev-validate e2e folder was an incorrect UUID. It needed to start with a 4 in the 3rd set of characters and had an extra character at the end.

The Composition test uses the remote location of  `https://raw.githubusercontent.com/defenseunicorns/lula/main/src/test/e2e/scenarios/dev-validate/validation.kyverno.yaml` to get the validation during compose so the composed artifact is invalid because of the UUID of the validation.

The bad UUID caused the error when the file is linted.

By changing the UUID it fixes the issues in the composed artifact which fixes the test that is ran by the compose_test.go which composed the OSCAL files in the src/test/unit/common/composition test directory.

## Related Issue

Fixes #
https://github.com/defenseunicorns/lula/issues/538 by updating the incorrect UUID in the file used for testing.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist before merging

- [ ] Test, docs, adr added or updated as needed
- [ ] [Schema Updates](https://github.com/defenseunicorns/lula/blob/main/docs/community-and-contribution/schema-updates.md) applied
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/lula/blob/main/CONTRIBUTING.md) followed

### CODE REVIEW

**Comment by CloudBeard** [comment]
E2E will fail until this fix is pushed to main.

https://github.com/defenseunicorns/lula/blob/575500e0ce40f786d2b6cdf143757e10e6e2c333/src/test/unit/common/composition/component-definition-local-and-remote.yaml#L43

The compose is testing local and remote OSCAL and this component-definition is looking at main and this change isn't there yet. Should be updated once and good to go.

Maybe pin a version once fixed and track with renovate? I think main is fine, shouldn't change much.

**Comment by brandtkeller** [comment]
> E2E will fail until this fix is pushed to main.
> 
> https://github.com/defenseunicorns/lula/blob/575500e0ce40f786d2b6cdf143757e10e6e2c333/src/test/unit/common/composition/component-definition-local-and-remote.yaml#L43
> 
> The compose is testing local and remote OSCAL and this component-definition is looking at main and this change isn't there yet. Should be updated once and good to go.
> 
> Maybe pin a version once fixed and track with renovate? I think main is fine, shouldn't change much.

There is a gap here for testing against `main` resources and doing feature development. Someone may unknowingly introduce a change to a file that is not picked-up until merge to main has happened. 

Some of this can be resolved with ensuring we have insights to valid testing files actually being valid (linted etc) and remaining that way. We should file an issue that looks at some method for classifying a file as a valid artifact (possibly filename identifier) and performs some linting to ensure they stay valid.

File: src/test/unit/common/composition/temp.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+assessment-results:
```
Comment (brandtkeller) [inline src/test/unit/common/composition/temp.yaml]: Was this meant to make it into the changes?

**Comment by brandtkeller** [comment]
Appreciate jumping to get this fixed. Only other comment I would make is additional context to assist reviews; you've got:
- What is the problem
- Where does it exist
- Why does it exist

The only missing piece of value is 'How to validate the changes'.

File: src/test/unit/common/composition/temp.yaml (hunk 1 above)
Comment (CloudBeard) [reply src/test/unit/common/composition/temp.yaml]: That needs removed. From a separate test that shouldn't be on this branch.

**Review by mike-winberry — APPROVED** [review]

**Comment by CloudBeard** [comment]
Verification of fix.

First screenshot is the fail in the E2E testing GitHub workflow. The test that is failing is the TestComponentDefinition in the `src/cmd/tools/compose_test.go`
 
image

Fix screenshots below show the `src/test/unit/common/composition/component-definition-local-and-remote.yaml` changed the url to this pr branch as the remote source and re-running the test specifically. (had some issues running make test but I believe thats something on machine vs the actual environment, more failing there than in CI)

image

**Review by brandtkeller — APPROVED** [review]

**Pull request merged**